### DISCUSSION TITLE
Add Custom value in wp-config.php

### DISCUSSION THREAD

**Discussion by chingman** [docker-library/wordpress#874] [General]
What is the best practice if I want to add / change the value inside the wp-config.php?

For example if I want to change:
/** Disable pseudo cron behavior. */
define('DISABLE_WP_CRON', true);
to 
/** Disable pseudo cron behavior. */
define('DISABLE_WP_CRON', false);

**Comment by yosifkit** [comment]
I don't see `DISABLE_WP_CRON` already defined in [`wp-config-docker.php`](https://github.com/docker-library/wordpress/blob/6e0d5acfc84bef260c85110da4b94d763ad5086c/wp-config-docker.php) and it isn't one of the env vars in the [Docker Hub readme](https://github.com/docker-library/docs/tree/9fa338f1c639b9db627e8e636e1b1e9fa97566ec/wordpress#how-to-use-this-image), so I would suggest using `WORDPRESS_CONFIG_EXTRA` for this (you provide it arbitrary PHP code that gets evaluated as part of `wp-config-docker.php`).

https://github.com/docker-library/wordpress/blob/6e0d5acfc84bef260c85110da4b94d763ad5086c/wp-config-docker.php#L121-L123
